linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml

```diff
@@ -0,0 +1,44 @@
+documentation_complete: true
+
+prodtype: rhel6,rhel7,rhel8,fedora,ol7
```

Comment:
Fedora should removed.

Reply:
We should remove fedora from anything related to FIPS.

Reply:
> I think if we remove Fedora from here now, the rules will be missing OVAL, because the dependent definition won't be there.

I think that as long as OVAL definition for `installed_OS_is_FIPS_certified` is marked as applicable for Fedora, it should be fine. This needs to be tested though.

> We should remove fedora from anything related to FIPS.

@redhatrises Do you consider it crucial that FIPS related rules are not listed in Fedora benchmarks?